esphome/core/defines.h

```diff
@@ -38,6 +38,10 @@
 #define USE_LIGHT
 #define USE_LOCK
 #define USE_LOGGER
+#define USE_LVGL
+#define LVGL_USES_FONT
+#define LVGL_USES_IMAGE
+#define LV_USE_TOUCHSCREEN
```

Comment:
These should follow the style of all others please =)
```suggestion
#define USE_LVGL_FONT
#define USE_LVGL_IMAGE
#define USE_LVGL_TOUCHSCREEN
```